src/sentry/static/sentry/app/components/searchBar.tsx

```diff
@@ -103,11 +105,15 @@ class SearchBar extends React.PureComponent<Props, State> {
             />
             <StyledIconSearch className="search-input-icon" size="sm" color="gray500" />
             {this.state.query !== this.props.defaultQuery && (
-              <div>
-                <a className="search-clear-form" onClick={this.clearSearch}>
-                  <IconClose />
-                </a>
-              </div>
+              <SearchClearForm
```

Comment:
I would rename for something like `SearchClearButton` but it's just a suggestion :)

Reply:
Good call 👍